sympy/combinatorics/free_group.py

```diff
@@ -806,6 +831,58 @@ def subword(self, from_i, to_j):
             array_form = letter_form_to_array_form(letter_form, group)
             return group.dtype(array_form)
 
+    def is_independent(self, word):
```

Comment:
This method is to be used to eliminate a few of the "relators" from the relator list. But I am not sure, about at what stage should this be used in the `elimination_technique_1`.